internal/flags/configure.go

```diff
@@ -38,11 +40,11 @@ func (f *Flags) handleConfigureCommand() utils.ReturnCode {
 	switch f.SubCommand {
 	case "addwifisettings":
 		rc = f.handleAddWifiSettings()
-		break
+	case "enablewifiport":
```

Comment:
did you mean to remove the breaks ?

Reply:
yes, unnecessary in Go and was being flagged in the editor.